platform/src/main/java/platform/pageobjects/Footer/FooterPage.java

```diff
@@ -9,141 +10,352 @@
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
-import platform.pageobjects.Authentication.LoginPage;
 import utils.CommonFunctions;
 
 /**
- * @Author : Chetan Sonparote
- * @Date of Creation : 26 Jun 2021
+ * @Author : Chetan Sonparote, Sahaj Balgunde
```

Comment:
Please create a new code block for authorship. dont merge into others

Reply:
Done the following change

Reply:
Please add your seperate authorship block for code changes that u have done